Vagrantfile

```diff
@@ -136,10 +136,10 @@ Vagrant.configure('2') do |config|
     if !Vagrant::Util::Platform.windows?
```

Comment:
Shouldn't this be enabled on Windows now that it no longer relies on bash? 🤔

Reply:
`~/.ssh/config` is a safe assumption on Windows as well?

Reply:
`%userprofile%/.ssh/config` in windows parlance, where `userprofile` is an environment variable to the user's home directory. while not all versions of windows come with openssh (windows 10 comes with it since earlier this year), git requires ssh, and so most windows users who are using trellis will probably have openssh and this is a safe assumption in those cases.

Reply:
And if that file exists, we can safely assume it's the same format I'm guessing?

Reply:
Yeah, it follows the same format, but one thing that would concern is me is whether the path to the private key is in the windows (e.g., `d:\Sites\example.com\trellis\etc...`) or unix (e.g., `/mnt/d/Sites/example.com/trellis/etc...`) format. There's probably not an easy way to handle this, but I'm thinking we could rely on Ruby/Vagrant to figure it out.

My Ruby is terrible, but I can test whatever you wanna send me.